corehq/toggles.py

```diff
@@ -220,14 +221,18 @@ def randomness_percent(self):
     def _get_identifier(self, item):
         return '{}:{}:{}'.format(self.namespaces, self.slug, item)
 
-    def enabled(self, item, **kwargs):
+    def enabled(self, item, namespace=Ellipsis):
```

Comment:
What do you think of changing the signature to `toggle.enabled("icds", domain=True)` or `toggle.enabled("dmiller@dimagi.com", user=True)` rather than requiring a `namespace` constant, which must be imported anywhere `toggle.enabled(...)` is called? We could require that the `domain` parameter is always specified as a keyword argument (using `**kwargs`) so we don't end up with ambiguous calls like `toggle.enabled("icds", True)`. If neither `domain` nor `user` is passed, then it would fall back to the existing behavior of checking if the flag is enabled for a (properly namespaced) user or domain with the given value.

This feedback can be ignored until later since it is less critical than the other comments on this PR.

Reply:
I really like this idea. I agree that its less pressing than the other changes, but let's not let it slide.